pkgs/servers/pulseaudio/default.nix

```diff
@@ -85,6 +85,7 @@ stdenv.mkDerivation rec {
     [ "--disable-solaris"
       "--disable-jack"
       "--disable-oss-output"
+      "--disable-neon-opt"
```

Comment:
Please only disable it on darwin. We also have aarch64-linux, where this probably works fine.

Reply:
Done